libs/partners/together/langchain_together/output_parsers.py

```diff
@@ -0,0 +1,94 @@
+import json
+from typing import Any, List, Optional
+
+import jsonpatch  # type: ignore[import]
+from langchain_core.exceptions import OutputParserException
+from langchain_core.output_parsers import BaseCumulativeTransformOutputParser
+from langchain_core.output_parsers.json import parse_partial_json
+from langchain_core.outputs import ChatGeneration, Generation
+
+
+class JsonOutputFunctionsParser(BaseCumulativeTransformOutputParser[Any]):
```

Comment:
is there anything this does that `langchain_core.output_parsers.openai_functions.JsonOutputFunctinosParser` doesn't already do?